docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
       command: |
         postgres
            -c wal_level=replica
-           -c synchronous_commit=remote_apply
+           -c synchronous_commit=off
```

Comment:
That is not an default behaviour of the postgres `synchronous_commit` setting. Default value is `on`, there might be some inconsistency with queries